keyboards/acheron/athena/gamma/info.json

```
@@ -0,0 +1,137 @@
+{
+    "keyboard_name": "Athena rev Gamma",
+    "usb": {
+        "pid": "0x6586",
+        "device_version": "0.0.1"
+    },
+    "matrix_pins": {
+        "rows": ["C14","C15","A3" ,"C10","A8" ,"C12","C6" ,"C9" ,"B14","C2" ,"C1" ,"C8" ,"B9" ],
+        "cols": ["C11","A2" ,"C3" ,"A0" ,"A1" ,"B10","B1" ,"B0" ,"C5" ,"C4" ,"A7" ,"A6" ,"A5" ,"A4" ,"A15"]
+    },
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "lto":       true,
+        "mousekey":  true,
+        "extrakey":  true,
+        "console":   false,
+        "backlight": true,
+        "rgblight":  true,
+        "audio":     false,
+        "encoder":   false,
+        "nkro":      true
+    },
+    "backlight": {
+        "pin": "C7",
+        "levels": 20,
+        "breathing": true
+    },
```

migrated from `config.h`
```suggestion
    },
    "rgblight": {
        "led_count": 33,
        "animations": {
            "breathing": true,
            "rainbow_mood": true,
            "rainbow_swirl": true,
            "snake": true,
            "knight": true,
            "christmas": true,
            "static_gradient": true,
            "rgb_test": true,
            "alternating": true,
            "twinkle": true
        }
    },
```